trials.js

```diff
@@ -0,0 +1,49 @@
+"use strict"; //Strict mode turn on
+// Md Mahadi Hasan Rifat
+// Breaking Record Hackerrank coding challange
+
+const breakingRecords = function (n) {
+  let maxScore = n[0];
+  let minScore = n[0];
+  let maxBroke = 0;
+  let minBroke = 0;
+  // for loop iteration for traverse all the arrays
+  for (let i = 0; i < n.length; i++) {
+    if (n[i] > maxScore) {
+      maxScore = n[i];
+      maxBroke++;
+    }
+
+    if (n[i] < minScore) {
+      minScore = n[i];
+      minBroke++;
+    }
+  }
+  return [maxBroke, minBroke];
+};
+
+console.log(breakingRecords([3, 4, 21, 36, 10, 28, 35, 5, 24, 42]));
+
+// Fred offei
+// Minimum Distances Challenge
+
+function minimumDistances(a) {
+  let distances = [];
```

Comment:
Irrelevant javascript code page makes your codebase confusing!
![DeleteButtonGIF](https://user-images.githubusercontent.com/123451855/235458427-26eafe79-69c5-4dfd-ac4e-2bd221f1ead1.gif)

Reply:
Good job commenting on this code 💯 , but it could be better to make our code more clear , it allows the student to know exactly hat to do , Kindly use the `It could be better` formula to explain to the student what they should do instead 😄 ,

Also, it could be better if you use `emoji` it will make your comments more friendly 😺